### PR TITLE
DAT-1193 - Fix Shopify page

### PR DESCRIPTION
**Pull request by jcamposmk**
### **Fix Shopify page**
**Jira Ticket:** https://makingsense.atlassian.net/browse/DAT-1193

**Before:**
Captura de Pantalla 2022-10-13 a la(s) 11 40 13 a  m

**After:**
Captura de Pantalla 2022-10-13 a la(s) 11 40 41 a  m

### CODE REVIEW

**Comment by andresmoschini** [comment]
Why the icons are broken?

File: src/utils.test.js [hunk 1]
```diff
@@ -161,73 +159,6 @@ describe('utils', () => {
...
-        }),
-      );
-      expect(counter).toBe(1);
-
-      jest.advanceTimersByTime(200);
-      // Assert
-      expect(counter).toBe(3);
-    });
-  });
-
```
Comment (andresmoschini) [inline src/utils.test.js:230]: Is possible to keep the tests? If I am not wrong, the behavior should be the same as before, right?

**Comment by jcamposmk** [comment]
> Why the icons are broken?

That image is in my local environment. Right now they look good. 
Captura de Pantalla 2022-10-13 a la(s) 12 31 32 p  m

File: src/utils.test.js (hunk 1 above)
Comment (jcamposmk) [reply src/utils.test.js:230]: Yes, the behavior should be the same as before. 
But, now I changed it to use a different `useInterval` located in `src/hooks/useInterval/index.js`. 
Practically it is a bit like the previous one, but it is a little more flexible.

Comment (andresmoschini) [reply src/utils.test.js:230]: > Practically it is a bit like the previous one, but it is a little more flexible.

But we loose tests :(

Comment (jcamposmk) [reply src/utils.test.js:230]: I'm going to upload a commit testing this functionality in the `Shopify Component`

File: src/components/Integrations/Shopify/Shopify.test.js [hunk 2]
```diff
@@ -205,4 +215,44 @@ describe('Shopify Component', () => {
...
+    // because it's executed to start
+    expect(getShopifyDataMock).toHaveBeenCalledTimes(1);
+
+    jest.advanceTimersByTime(FETCH_SHOPIFY_DATA_INTERVAL);
+    expect(getShopifyDataMock).toHaveBeenCalledTimes(2);
+
+    jest.advanceTimersByTime(FETCH_SHOPIFY_DATA_INTERVAL);
+    expect(getShopifyDataMock).toHaveBeenCalledTimes(3);
+  });
 });
```
Comment (jcamposmk) [inline src/components/Integrations/Shopify/Shopify.test.js:258]: This was the missed test @andresmoschini

**Comment by jhoffman-ms** [comment]
👍

File: src/components/Integrations/Shopify/Shopify.test.js (hunk 2 above)
Comment (andresmoschini) [reply src/components/Integrations/Shopify/Shopify.test.js:258]: 👏👏👏

**Pull request merged**